test/makeRule.js

```diff
@@ -427,6 +427,7 @@ const parserOptions = {
             greetings: () => Relay.QL\`
               fragment on Greetings {
                 hello,
+                hi,
```

Comment:
I added this here to validate that the rule is not reporting when it's not enabled.